### PR TITLE
Lgbtq variations

### PR DESCRIPTION
**Pull request by KathyReid**
There are LGBTQI variations on the Linux Australia logo. They are intended to show support for, and allyship with, our LGBTQI community. They are done as SVG with PNG previews so that they are vectorised, for example printing stickers with. 

I'd like to seek feedback from the LGBTQI community first before sending this to Linux Aus Council for approval.

### CODE REVIEW

**Review by mattcen — CHANGES_REQUESTED** [review]
To adhere with suggestions at https://en.wikipedia.org/wiki/Bisexual_pride_flag#Design_and_colors, I recommend these colour substitutions. The current blue looks far too purple and not resemblant of the Bi+ flag to me. (I understand the decision for the current colours may have a good reason, such as adherence to a larger style, or good contrast etc., but it *really* doesn't jump out to me as the Bi+ flag... and I usually see them *everywhere* :P)
`sed -i 's/#3a00a8/#0038a8/g; s/#ad0d94/#9b4f96/g; s/#ef0069/#d60270/g' linuxaus-bi.svg`

Further, this looks great as a Pansexual flag if you want one of those:
`sed  's/#3a00a8/#21b1ff/g; s/#ad0d94/#ffd800/g; s/#ef0069/#ff218c/g' linuxaus-bi.svg > linuxaus-pan.svg`

Further, the white outer circles of the Trans flag don't have borders between them and the large inner circle, unlike the Enby and Ace versions. I think the Enby and Ace versions look better having this grey border. I couldn't work out how to make Inkscape do this for me, unfortunately, or I'd have sent a diff.

The LGBTQ flag colours don't match the traditional ones, but I like them and it's still recognizable. within context.

I think the rest look pretty good and recognizable, but not being Ace, Trans, or Enby myself, I feel less comfortable speaking for those ones. All I can see is I can easily recognise them and they look pretty close to the representations on [Wikipedia](https://en.wikipedia.org/wiki/LGBT_symbols), without me comparing the RGB values with the Wiki source SVGs.

**Comment by mattcen** [comment]
I worked out how to update the Trans flag circles, and have bundled my proposed changes into #5.

**Comment by daisylb** [comment]
Speaking as someone who is bi, non-binary, and transgender: my main concern would be the intended usage. Printing on stickers and similar sounds fine, other usages I'd make sure that at least somebody to whom the relevant flag applies is involved in whatever thing it's being used for.

Aesthetically, I'm wondering whether the stroke on the white circles in the trans and nb ones are going to be too thin, depending on the way in which it's used. (I think they'd definitely be too thin for small stickers.)

**Comment by KathyReid** [comment]
I'd also like to note feedback from @SuperRoach provided to me privately around the `stroke-weight` on the images which have white circles, such as the _enby_ image - essentially if the images are scaled down, the `stroke-weight` may be so thin that it disappears. 

To try and test this, I scaled down one of the images to see whether the `stroke-weight` becomes too thin when the image is scaled down: 

linuxaus-ace svg

I agree, I think the `stroke-weight` is too thin (thanks too to @excitedleigh for your feedback here too)

Next action: @KathyReid to do a separate commit which iterates the `stroke-weight`

**Comment by KathyReid** [comment]
Picking up on @excitedleigh's point about the _use_ of these images - does it make sense to make this repo private to Linux Australia folx only? This would stop the images being used without consideration, but detracts from their openness.

**Comment by KathyReid** [comment]
I've now modified the `stroke-width` on the images that have white circles in commit `dbcd419`. 
As an example, this is how the ace image now appears when shrunk: 

linuxaus-ace-shrunk

I think the `stroke-width` is a lot clearer.

**Comment by KathyReid** [comment]
I'd also like to acknowledge feedback from @skraynick that was provided privately - around the circle edges not showing on circles that are white. @mattcen has already kindly addressed this :-) 

Thanks so much @skraynick for your feedback - very much appreciated.

**Comment by daisylb** [comment]
Re making the repo private: IMO unauthorised use of the logo doesn't seem like it's going to be an issue; I'm more concerned about making sure that people representing LA consider how and where they use them. Maybe some usage guidelines to go along with them?

**Comment by KathyReid** [comment]
Apologies for letting this one sit, I've been in transit back to Geelong from Canberra. 

There were two pieces here that I think were unfinished; 

* _Trying to have the flags in colour order sequence_: This suggestion was made [on Twitter](https://twitter.com/EricGershman) by @EricGershman (sorry, Eric, for taking a while to reply). I've now done a version based on this suggestion for comment and feedback: 

linuxaus-lgbtqi-sequence-reduced

My personal feeling is that the _sequence_ of colours better represents the flag, however I'd really like to seek feedback on this. 

*_Usage guidelines for these logo variants_: @excitedleigh makes a very good point about usage guidelines. 

How does the below sound: 

```
Diverse logo representation for Linux Australia - usage guidelines 

Diverse representations of the Linux Australia logo have been created with the express purpose of allyship, representation and visibility for our LBGTQI+ community. 

You are allowed to use these logos for these purposes without consulting with Linux Australia; 
* stickers,
* badges, 
* pins, 
* and similar artefacts 

Please consider the intent and context in which you use these artefacts, and be mindful of their use being interpreted as satire, mockery or in a pejorative manner. 

All other uses of the logo should be referred to council@linux.org.au for discussion. 

```

Best, Kathy

**Comment by daisylb** [comment]
I think the in-sequence variants are much better; it's much clearer what's being represented! I also think the usage guidelines sound reasonable.

**Comment by KathyReid** [comment]
Next actions for me: 

* Turn the images into colour-sequential ones 
* Add the usage guidelines into the README 
* Send this to Council for consideration / approval

**Comment by KathyReid** [comment]
Here are the logos with colours in sequence - thumbnails to save folx having to open up a bunch of files :-) 

linuxaus-ace-small
linuxaus-bi-small
linuxaus-enby-small
linuxaus-genderfluid-small
linuxaus-lgbtqi-small
linuxaus-pan-small
linuxaus-transgender-small

README has been updated with Usage Guidelines. 

Now sending to Linux Aus Council for approval. 

A huge thanks everyone for your excellent feedback, contributions, constructive comments and wisdom. Together, we really do do better :100:

**Review by M0les — APPROVED** [review]
Looks fab, well considered and inclusive.

**Comment by KathyReid** [comment]
From @jwoithe on behalf of LA Council 

```
Hi Kathy

On Tue, Dec 03, 2019 at 08:00:11PM +1100, Kathy Reid via committee wrote:
> TL;DR I've done some variations of the LA logo and matched it to various
> LGBTQI+ preferences, sought community feedback, iterated, and am at a point
> where I'd like to get signoff from Council on the designs, with a view to
> getting stickers printed for conf.
>
> Feedback, as always, warmly welcomed.
>
> https://github.com/linuxaustralia/brand-guidelines/pull/4

Council has voted on the logo derivatives and has approved them unanimously. 
We apologise that it took some time to reach this point.  Thanks for your
work on both the logo itself and the associated community engagement.

This means that you are free to use these logos on stickers for LCA2020.

FYI this work has brought up the issue of logo derivatives.  It is felt by
some that we probably need to draw up some simple guidelines so the
community is clear about what derivative works are considered acceptable and
what uses they can be put to.  However, given the time of year this would be
something for the incoming council to act on as they see fit.

Regards
```
Merging PR on this basis, many thanks @M0les for reviewing & @jwoithe for representing at Council.
